reco_utils/recommender/rlrmc/pymanopt/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .core.problem import Problem

```

Comment:
If pymanopt can be installed via pip, it would be probably better to add it in the conda installation. Here there is the file: https://github.com/Microsoft/Recommenders/blob/staging/scripts/generate_conda_file.py#L67, for most libraries we tend to use `==` instead of `>=` when setting the version

Reply:
Yes, I checked that this suggestion works and we should go ahead with this.